src/Textures.cpp

```diff
@@ -1015,7 +1015,7 @@ void TextureCache::_loadBackground(CachedTexture *pTexture)
 		params.width = pTexture->realWidth;
 		params.height = pTexture->realHeight;
 		params.format = colorFormat::RGBA;
-		params.internalFormat = glInternalFormat;
+		params.internalFormat = gfxContext.convertGHQTextureFormat(u32(glInternalFormat));
```

Comment:
> Those two seem ok, but this contradicts:

You misunderstood me. Your fix replaces converts any internal formats, including GL_RGB5_A1, GL_RGBA4 to GL_RGBA. However, internal structure of texture data does not changed. It still will be 5551 or 4444. I mean that GLES2 works with these textures correctly because it has DataType information, as GL_UNSIGNED_SHORT_5_5_5_1 and it is enough.

Currently, we have the following internal color formats, which GLES2 code can met:
```
		InternalColorFormatParam RGB8(GL_RGB);
		InternalColorFormatParam RGBA8(GL_RGBA8);
		InternalColorFormatParam RGBA4(GL_RGBA4);
		InternalColorFormatParam RGB5_A1(GL_RGB5_A1);
		InternalColorFormatParam LUMINANCE(0x1909);

```
I suggest, instead of 
`params.internalFormat = gfxContext.convertGHQTextureFormat(u32(glInternalFormat));`
which is contradicts with semantic of gfxContext.convertGHQTextureFormat function,
replace internal RGB format by GL_RGB and RGBA formats by GL_RGBA in ContextImpl::init() for GLES2.